Source/WebCore/css/CSSCounterStyle.cpp

```diff
@@ -236,6 +237,11 @@ static String counterForSystemCJK(int number, const std::array<UChar, 17>& table
     return std::span<const UChar> { characters, length };
 }
 
+String CSSCounterStyle::counterForSystemDisclosureClosed(TextDirection textDirection)
+{
+    return textDirection == TextDirection::LTR ? span(blackRightPointingSmallTriangle) : span(blackLeftPointingSmallTriangle);
```

Comment:
What is the `span` for ?

Reply:
We don't have a String constructor for UCHAR IIRC

Reply:
Yeah, we have removed String constructors that take raw char ptrs here: https://commits.webkit.org/276859@main